tests/Melodiia/Bridge/Symfony/Form/DateTimeTypeTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Biig\Melodiia\Test\Bridge\Symfony\Form;
+
+use Symfony\Component\Form\Test\TypeTestCase;
+
+class DateTimeTypeTest extends TypeTestCase
+{
+    public const TESTED_TYPE = 'Biig\Melodiia\Bridge\Symfony\Form\Type\DateTimeType';
+
+    public function testItChangesDefaultOptions()
+    {
+        $form = $this->factory->create(self::TESTED_TYPE, null, [
```

Comment:
Why don't use directly the FQCN of the class?

And you can also remove the optional `options` parameter, that will avoid the new line 👍

Reply:
Well, it's done like that in Symfony. But they use another class than `TypeTestCase`. I change it for us.